src/modules/ganalytics/AnalyticsProvider.tsx

```diff
@@ -1,107 +1,177 @@
-import React, { useContext, useEffect } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { useWallet } from '@solana/wallet-adapter-react';
 import { Coingecko, Currency } from '@metaplex/js';
 import { WalletContext } from '@/modules/wallet';
+import Bugsnag from '@bugsnag/js';
+import BugsnagPluginReact from '@bugsnag/plugin-react';
+
 import { Listing } from '@/modules/indexer';
+import { useRouter } from 'next/router';
+import {
+  getFormatedListingPrice,
+  lamportToSolIsh,
+} from '@/common/components/elements/ListingPreview';
 
-export const GOOGLE_ANALYTICS_ID = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS_ID || 'G-HLNC4C2YKN';
+export const OLD_GOOGLE_ANALYTICS_ID = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS_ID;
```

Comment:
I'm for making a clean switch. Just say on the release date all data will now be going to update google tracking.

Reply:
There is no penalty for running it in parallel. I guess I'm a bit afraid of something not working in prod and us losing data from the initial/old implementation.

I can make a ticket and a PR for moving "storefront builder" analytics to the new provider after this one is merged. Would that work for you @kyle? :)